src/test/java/com/egov/core-services/constants/eGovWorkFlowProcessSearch.yaml

```diff
@@ -0,0 +1,10 @@
+parameters:
+ invalidTenantId: abc
```

Comment:
invalidTenantId, multipleTenantId and invalidAuthToken can be moved to commonConstants.yaml and reused across other modules also